src/components/shared/field.svelte

```diff
@@ -0,0 +1,119 @@
+<script>
+import BSN from "bootstrap.native/dist/bootstrap-native.esm.min.js";
+import { createEventDispatcher, onMount } from "svelte";
+
+const dispatch = createEventDispatcher();
+
+// Props
+export let item = {};
+export let value = null;
+
+// Auxiliary variables
+let dropdown = null;
+
+// Bindings
+let dropdownButton;
+let popoverIcon;
+
+// Computed variables
+const isCheckbox = item.type === "checkbox";
+const isInput = item.type === "input";
+const isDropdown = item.type === "dropdown";
```

Comment:
I haven't yet tried computed variables, but _I think_ you should be doing `$: isDropdown = item.type === "dropdown"` (source: https://svelte.dev/tutorial/reactive-declarations).

> Your code probably works because you aren't changing `item.type` at runtime.